.gitignore

```diff
@@ -1,2 +1 @@
 qemu-binaries/
-rootfs.cpio
```

Comment:
doesn't this mean then that each time we decompress an initrd, that the directory is now dirty?

Reply:
No, I just moved it to `images/.gitignore`.